### ISSUE TITLE
Change in global_customization repo will trigger only 2 pipelines (out of 30)

### ISSUE THREAD

**Issue by damovsky** [aws-ia/terraform-aws-control_tower_account_factory#143]
**AFT Version:**
AFT Version:
1.3.5

**Bug Description**
3 days ago I made a change in aft-global-customizations repo. I pushed the change into AWS CodeCommit.
Results:
- only two pipelines had been triggered (out of 30 existing)
- when I clicked to "Release change" on the pipeline page, the pipeline gets triggered and update the aws account with latest changes from global customization repo...so now I have to click to "release change" on every pipeline manually

It happens to me multiple times....I haven't touch the pipeline settings - it uses the default CloudWatch events Change Detection option.

My analysis:
- cloudwatch event rule exists
- the target for the cloudwatch rule contains only 2 pipelines.
- I would expect there  30 pipelines as a target and not 2...

**Expected behavior**
Everytime I push a change into global customization repo, ALL pipelines are triggered in order to update ALL AWS accounts

**Comment by snebhu3** [comment]
@damovsky thank you for reaching out. When you commit changes to aft-global-customizations, the individual vended account pipelines `<vended-account-id>-customizations-pipeline ` do not get triggered automatically, which is an expected behavior. 

After the aft-global-customizations repo change, you would need to trigger the `aft-invoke-customizations` step function with required targets to invoke the required `<vended-account-id>-customizations-pipeline`. You could target all accounts, or specific accounts based on your roll-out strategy. Please refer to [documentation](https://docs.aws.amazon.com/controltower/latest/userguide/aft-account-customization-options.html#aft-global-customizations) for more details.

However, I am curious as to which 2 pipelines were triggered when you pushed a change to  aft-global-customizations repo, please may you provide more details here?

**Comment by damovsky** [comment]
@snebhu3 thanks for quick response!

now it makes perfectly sense! and I have an explanation for those 2 pipelines. They were triggered "thanks to misleading" aws behaviour.

What I did:

1. open one of `<vended-account-id>customization-pipeline`
2. click to `EDIT`
3. click to `Edit stage` for fist stage `Edit: Source`
4. click to edit button for `aft-global-customizations`, see image 
![Screenshot 2022-04-26 at 8 20 31](https://user-images.githubusercontent.com/8916256/165234896-72e5ec5e-3bb0-4990-84c2-77f9c732ad4c.png)

5. **do nothing** and click to Done
6. Click to Save
7. Dialog to confirm saving changes appears - despite of fact I did not make any changes....and here is the issue: here it will add cloudwatch event rule
![image](https://user-images.githubusercontent.com/8916256/165235283-d28122ea-4e10-442e-b623-11151b102b04.png)

and yes, I should read all the dialogs more carefully next time, mea culpa.

This is really annoying and stupid behavior.  Saving pipeline configuration without any change made by user will introduce this CloudWatch event rule to trigger pipeline upon git push.
